subsys/caf/modules/net_state_lte.c

```diff
@@ -8,34 +8,37 @@
 #include <zephyr.h>
 #include <modem/lte_lc.h>
 
-#include "net_event.h"
-
-#define MODULE net
+#define MODULE net_state_lte
```

Comment:
Same for `net_state_ot.c`

Reply:
I noticed it too. But there is one advantage we gain from having it and that is being able to have information from two networks.
At the moment nothing is checking on module net so splitting the name will not affect us. On the other hand we also don't have case with simultaneous OT and LTE. I thought that the later would be more useful in the end as even though  we probably won't get OT&LTE on a single board we might get two other networks (e.g. BLE & OT).

It is possible that I was wrong and at least for now we should allow only one net module to be present.

The problem you mentioned is universal and we should think about resolving it for any family of modules. I.e. what to do in case two modules from the same family coexist? Should we wait for all modules or only for a single one? How to group family together?

Reply:
@rakons let's keep the same name "net_state" for both instances for now. Please switch to choice in Kconfig so it won't be possible to enabled both.

Reply:
I was providing it that way to allow them both to work together. What if we would support wifi in the future?

Reply:
Notice that the options that we are discussing are not settable by the user. I was trying to convert it to choice but choice requires all the options to have a prompt. This are internal options that are set depending on the other symbols.

Reply:
Fixed as discussed offline - common modules names but no change in Kconfig.